bschweiz/HappyPlants
# Dec9 auth login

# Description
this pull requests brings in working user Login and Registration components. next on the list is building out NavBar functionality and basic renderings of the Event and Plant List pages

Fixes # (user login/registration)
## Type of change
Please delete options that are not relevant.
- [] Bug fix (non-breaking change which fixes an issue)
- [] Breaking change (fix or feature that would cause existing functionality to not work as expected)
# Testing Instructions
make sure local storage is clear of user id's and reload page, can create a new user and/or login with pre-existing user 
# Checklist:
- [x ] My code follows the style guidelines of this project
- [ x] I have performed a self-review of my own code
- [ x] I have commented my code, particularly in hard-to-understand areas
- [x ] My changes generate no new warnings or errors
- [ x] I have added test instructions that prove my fix is effective or that my feature works
